2017_01_04_word_count/wc.js

```diff
@@ -0,0 +1,23 @@
+// Run this with:
+// node wc.js quotes.txt
+var fileName = process.argv[2];
+fs = require('fs')
+fs.readFile(fileName, 'utf8', function (err, data) {
+    if (err) {
+        return console.log(err);
+    }
+
+    console.log(lineCount(data) + ' ' + wordCount(data) + ' ' + totalChars(data) + ' ' + fileName);
+});
+
+function lineCount(data) {
```

Comment:
Found one after some quick Googling actually. TIL. Thanks Clay.

http://adripofjavascript.com/blog/drips/variable-and-function-hoisting